packages/fluentui/react-builder/src/components/ReaderText.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Alert, Ref } from '@fluentui/react-northstar';
+
+export type ReaderTextProps = {
+  selector: string;
+};
+
+export const ReaderText: React.FunctionComponent<ReaderTextProps> = ({ selector }) => {
+  const ref = React.createRef<HTMLElement>();
+  const [text, setText] = React.useState('');
+
+  React.useEffect(() => {
+    if (ref.current) {
```

Comment:
Should `selector` be part of the condition?

Reply:
it could, but since this is typescript, it should be checked during build time, I hope